ci/install.sh

```diff
@@ -10,7 +12,7 @@ main() {
         sort=gsort  # for `sort --sort-version`, from brew's coreutils.
     fi
 
-    # Builds for iOS are done on OSX, but require the specific target to be
+    # Builds for iOS are done on macOS, but require the specific target to be
```

Comment:
Here I'd rather keep the file as is and add a reference to https://github.com/japaric/trust which it originates from

Reply:
Fixed at 93ef21f